pattern-1/dockerfiles/apim-analytics-1/carbon/bin/load-spark-env-vars.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+
+echo 'Loading spark environment variables '
+export CARBON_SPARK_HOME=$CARBON_HOME
+export _SPARK_ASSEMBLY=$CARBON_SPARK_HOME/repository/components/plugins/spark-core_2.10_*.wso2*.jar
+export SPARK_SCALA_VERSION=2.10
+# *** jars will be added to the spark classpath in the code itself. check DAS-105
+# export SPARK_CLASSPATH=`java -cp $CARBON_SPARK_HOME/repository/components/plugins/org.wso2.carbon.analytics.spark.utils*.jar org.wso2.carbon.analytics.spark.utils.ComputeClasspath $CARBON_HOME`
+# export SPARK_CLASSPATH=$SPARK_CLASSPATH:$(echo $CARBON_SPARK_HOME/repository/components/lib/*.jar | tr ' ' ':')
```

Comment:
Its not required now; removed with commit 6e116233b78b67bff454f3a1ba96927fbd6cafbd.